pkg/aggregator/aggregator.go

```diff
@@ -17,7 +17,7 @@ import (
 	"github.com/DataDog/datadog-agent/pkg/serializer/split"
 	"github.com/DataDog/datadog-agent/pkg/tagger"
 	"github.com/DataDog/datadog-agent/pkg/tagger/collectors"
-	"github.com/DataDog/datadog-agent/pkg/tagset"
+	oldtagset "github.com/DataDog/datadog-agent/pkg/tagset/old"
```

Comment:
Let's make new tagset impl `pkg/tagset/v2` and leave old usage intact until we are really ready to change it. WDYT?